students/swietczak_monika/lesson_05_lists/character_picture_grid.py

```diff
@@ -0,0 +1,16 @@
+grid = [['.', '.', '.', '.', '.', '.'],
+        ['.', 'O', 'O', '.', '.', '.'],
+        ['O', 'O', 'O', 'O', '.', '.'],
+        ['O', 'O', 'O', 'O', 'O', '.'],
+        ['.', 'O', 'O', 'O', 'O', 'O'],
+        ['O', 'O', 'O', 'O', 'O', '.'],
+        ['O', 'O', 'O', 'O', '.', '.'],
+        ['.', 'O', 'O', '.', '.', '.'],
+        ['.', '.', '.', '.', '.', '.']]
+
+for x in range(6):
+    for y in range(9):
+        if y < 8:
+            print(grid[y][x], end='')
+        else:
+            print(grid[y][x])
```

Comment:
I would avoid using the additional keyword argument 'end' in print method if it's not necessary. It's also not supported by python < 3

Consider the following example:
```
for x in range(len(grid[0])):
    line = ""
    for y in range(len(grid)):
        line += grid[y][x]
    print(line)
```